tictactoe/implementation/src/test/kotlin/com/anaplan/engineering/azuki/tictactoe/implementation/TestGame.kt

```diff
@@ -0,0 +1,36 @@
+package com.anaplan.engineering.azuki.tictactoe.implementation
+
+import org.junit.Test
+import kotlin.test.assertEquals
+
+class TestGame {
+
+    @Test
+    fun emptyBoard() {
+        val player1 = Player("1", Token.Cross)
+        val player2 = Player("1", Token.Circle)
```

Comment:
⛏️ I get it that only the token matters for the test, but...
```suggestion
        val player2 = Player("2", Token.Circle)
```

Reply:
I haven't touched this

Reply:
Indeed, it was my mistake. 😛